Dockerfile

```diff
@@ -0,0 +1,30 @@
+# Author: Supreeth Suresh
+# Affiliation: National Center for Atmospheric Research, Computational and Information Systems Laboratory
+# Created: 03-29-2024
+
+FROM ssuresh1809/samurai:latest
+
+LABEL maintainer="Supreeth Suresh <ssuresh@ucar.edu>"
+
+RUN mkdir -p /app/
+WORKDIR /app/
+RUN git clone https://github.com/mmbell/samurai.git && \
+    cd samurai && \
+    sed -i 's/MODE GPU/MODE CPU/g' CMakeLists.txt && \
+    mkdir build && \
+    cd build && \
+    cmake .. && \
+    make -j 2
+
+ENV PATH="/app/samurai/build:${PATH}"
+ENV LD_LIBRARY_PATH="/usr/local/lib:${LD_LIBRARY_PATH}"
+ENV PATH="/usr/local/bin:${PATH}"
+
+RUN echo "#!/bin/bash \n\
+cd /app/samurai/build/release/bin/ \n\
+./samurai /app/data/beltrami_preprocessed.xml\
```

Comment:
Samurai is now using the TDRP configuration files.  I have placed the TDRP files in the ncar_scripts/TDRP directory.  The equivalent command to run the beltrami testcase is now the following 

./samurai -params beltrami.tdrp

Reply:
This will require Supreeth to upload the TDRP file to his image so that I could update the run command as suggested here.

Reply:
My bad. The data comes with SAMURAI source code and is not need to be copied manually.